INSTALL.rst

```diff
@@ -52,11 +52,49 @@ results, you will additionally need:
 * The Slater-Koster data used in the tests (see below)
 
 
+Successful builds
+-----------------
+
+DFTB+ is regularly built and tested in following build environments:
+
++---------------+----------------------+-------------+------------------+-----+
+| Architecture  | Compiler             | MPI         | Ext. libraries   |Notes|
++===============+======================+=============+==================+=====+
+| x86_64 /      | GNU Fortran/C 7.5    | OpenMPI 2.1 | OpenBlas 0.3.7,  |     |
+| Linux         |                      |             | ScaLAPACK 2.1    |     |
++---------------+----------------------+-------------+------------------+-----+
+| x86_64 /      | GNU Fortran/C 10.1   | OpenMPI 4.0 | OpenBlas 0.3.10, |     |
+| Linux         |                      |             | ScaLAPACK 2.1    |     |
++---------------+----------------------+-------------+------------------+-----+
+| x86_64 /      | Intel Fortran/C 18.0 | MPICH 3.2   | MKL 18.0         |     |
+| Linux         |                      |             |                  |     |
++---------------+----------------------+-------------+------------------+-----+
+| x86_64 /      | Intel Fortran/C 18.0 | MPICH 3.2   | MKL 18.0         |     |
+| Linux         |                      |             |                  |     |
++---------------+----------------------+-------------+------------------+-----+
+| x86_64 /      | Intel Fortran/C 19.0 | MPICH 3.3   | MKL 19.0         |     |
+| Linux         |                      |             |                  |     |
++---------------+----------------------+-------------+------------------+-----+
+| x86_64 /      | NAG Fortran 7.0      | MPICH 3.3   | OpenBlas 0.3.7   |     |
+| Linux         | GNU C 9.2            |             | ScaLAPACK 2.1    |     |
++---------------+----------------------+-------------+------------------+-----+
+| x86_64 /      | GNU Fortran/C 8      | --          | LAPACK / BLAS    | [1] |
+| OS X          |                      |             |                  |     |
+|               |                      |             |                  |     |
++---------------+----------------------+-------------+------------------+-----+
+
+All builds use ARPACK-NG 3.7, ELSI 2.6.1 and PLUMED 2.5.
```

Comment:
All builds are also tested with the optional ARPACK-NG 3.7, ELSI 2.6.1 and PLUMED 2.5 libraries.